client/src/components/RatingsAndReviews/AddReviewModal.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+class AddReviewModal extends React.Component {
+  constructor(props) {
+    super(props);
+  };
+
+  componentDidMount() {
+    document.getElementById('default-modal-close-button').addEventListener('click', (event) => {
+      this.props.top.hideModal();
+    })
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        <h1>Example Modal Window</h1>
+        <form id="review-upload-form"
+          data-testid="review-upload-form"
+          encType="multipart/form-data"
+          action="/photo-upload"
+          method="post">
+          <input type="file" name="review-photo" multiple />
+          <input type="submit" value="Upload Image" name="submit" data-testid="review-upload-form-submit" />
+          <input type='text' id='random' name='random' /><br></br>
+          <span id = "status"></span>
```

Comment:
i wanna delete those spaces so bad, but not that important
```suggestion
          <span id="status"></span>
```